src/mctpd.c

```diff
@@ -1666,6 +1697,164 @@ static int message_read_hwaddr(sd_bus_message *call, dest_phys* dest)
 	return 0;
 }
 
+static int setup_static_eid(ctx *ctx)
+{
+	int rc;
+	sd_bus_error error = SD_BUS_ERROR_NULL;
+	sd_bus_message *message = NULL;
+	char **s;
+
+	rc = sd_bus_call_method(ctx->bus,
+        "xyz.openbmc_project.ObjectMapper", "/xyz/openbmc_project/object_mapper",
```

Comment:
Minor: Indentation is a little off here.

Also, might be worthwhile adding `const`s or `#define`s for these bus names / object paths / interfaces.

Reply:
Done